resources/net.unvanquished.Unvanquished.desktop

```diff
@@ -9,5 +9,16 @@ Exec="%1/updater" -- %u
 TryExec=%1/updater
 MimeType=x-scheme-handler/unv
 Categories=Game;ActionGame;StrategyGame;
+StartupWMClass=net.unvanquished.Unvanquished
+SingleMainWindow=true
 # Probably doesn't work since the updater is initially launched, not daemon
 PrefersNonDefaultGPU=true
+Actions=Run;Update;
+
+[Desktop Action Run]
```

Comment:
Please move the Actions thing to a separate PR. I will need to check whether it really can work correctly to use the internalcommand stuff at the top level.